### PR TITLE
Corrects a typo in filters names in usage section

### PR DESCRIPTION
**Pull request by Guilhermeasper**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,9 +19,9 @@ You can contact us for support on our [chat server](https://discord.gg/V3vSCs7)
 const ytsr = require('ytsr');
 let filter;
```
Comment (TimeForANinja) [inline README.md:20]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -19,9 +19,9 @@ You can contact us for support on our [chat server](https://discord.gg/V3vSCs7)
...
 
-ytsr.getFilters('github').then(async filters1 => {
-  filter1 = filters.get('Type').find(o => o.name === 'Video');
-  const filters2 = await ytsr.getFilters(filter.ref);
+ytsr.getFilters('github').then(async (filters1) => {
+  filter1 = filters1.get('Type').find(o => o.name === 'Video');
+  const filters2 = await ytsr.getFilters(filter1.ref);
   filter2 = filters2.get('Duration').find(o => o.name.startsWith('Short'));
```
Comment (TimeForANinja) [inline README.md:25]:
```suggestion
  const filter1 = filters1.get('Type').find(o => o.name === 'Video');
  const filters2 = await ytsr.getFilters(filter1.ref);
  const filter2 = filters2.get('Duration').find(o => o.name.startsWith('Short'));
```

**Review by TimeForANinja — COMMENTED** [review]
thanks for spotting
since were already at it...

**Comment by Guilhermeasper** [comment]
> thanks for spotting
> since were already at it...

You're welcome, I thank you for creating such a useful library.
I updated the version of ytsr in my project and as callbacks became deprecated, I came to see what the new way of performing searches was like and then I noticed the small typo.

**Pull request merged**